### ISSUE TITLE
consume() method is missing from channelWrapper

### ISSUE THREAD

**Issue by syberkitten** [benbria/node-amqp-connection-manager#67]
Looks like the channelWrapper is missing the consume method.

Only the inner AMQP channel has the consume method, but 
it is not available on the wrapper, is this by design?

Looks like it can be accessed via:
channelWrapper._channel.consume

same goes for "reject" which is also missing.

**Comment by syberkitten** [comment]
The issue is really that i need to have the ChannelWrapper inside the consume
message method so it would be possible to ack / reject messages.

The problem is that the ChannelWrapper is created after all the channel asserts / consume
are set up thus it's not available, which causes many side effects.

when i try to work around it, the re-connect feature is not working properly,
the connection re-connects but the channel is not re-created so messages are not received.

**Comment by FINDarkside** [comment]
ChannelWrapper will be created before your setup function is called for the first time. This means you can use channelWrapper inside the setup function if you need to.

**Comment by reachmehere89** [comment]
I agree to have consume method on channelwrapper  which would make calling consume method outside of setup
